00-max.js

```diff
@@ -1,3 +1,11 @@
 export function max(array) {
-    // TODO: implement max function
+    let output;
+    let isArrayHaveContent = checkArrayContent(array);
+    isArrayHaveContent == true ?  output = Math.max.apply(Math.max,array) :  output = undefined;
```

Comment:
`booleanValue == true` can be written in a simpler way: `booleanValue`. If you know that's a boolean, there's no point in comparing it to another value.